pages/es/supported-networks/cosmos.mdx

```diff
@@ -0,0 +1,191 @@
+
+## What are Cosmos subgraphs?
+
+Cosmos subgraphs are a new type of subgraph that [Graph Node](https://github.com/graphprotocol/graph-node) is now able to process. This allows to index and store data inside the Cosmos Hub blockchain, and make that resulting data easily available via GraphQL API.
+
+Cosmos subgraphs will differ from other implementations, such as Ethereum, in that there is no need to specify a smart contract from which events and data will be indexed from. In this case, Graph Node is monitoring all data appended to the chain.
+
+There are currently three types of handlers supported for the Cosmos Hub subgraphs:
+
+- Block handlers.
+- [Event](https://docs.cosmos.network/master/core/events.html) handlers.
+- [Transaction](https://docs.cosmos.network/master/core/transactions.html) handlers.
+
+Based on the [official Cosmos documentation](https://docs.cosmos.network/):
+
+> Events are objects that contain information about the execution of the application. They are mainly used by service providers like block explorers and wallet to track the execution of various messages and index transactions.
+
+> Transactions are objects created by end-users to trigger state changes in the application.
+
+## Building a Cosmos Hub subgraph
+
+### Subgraph Dependencies
+
+[graph-cli](https://github.com/graphprotocol/graph-cli) is a CLI tool to build and deploy subgraphs, version `>=0.23.0` is required in order to work with Cosmos Hub subgraphs.
+
+[graph-ts](https://github.com/graphprotocol/graph-ts) is a library of subgraph-specific types, version `>=0.23.0` is required in order to work with Cosmos Hub subgraphs.
+
+### Subgraph Main Components
+
+There are three main key parts or files when it comes to define a subgraph:
+
+**subgraph.yaml**: a YAML file containing the subgraph manifest.
+
+**schema.graphql**: a GraphQL schema that defines what data is stored for your subgraph, and how to query it via GraphQL.
+
+**AssemblyScript Mappings**: [AssemblyScript](https://github.com/AssemblyScript/assemblyscript) code that translates from the event data to the entities defined in your schema.
+
+### Subgraph Manifest Definition
+
+The subgraph manifest (`subgraph.yaml`) identifies the data sources for the subgraph, the triggers of interest, and the functions that should be run in response to those triggers. See below for an example subgraph manifest for a Cosmos Hub subgraph:
+
+```yaml
+specVersion: 0.0.2
+description: Cosmos Subgraph Example
+schema:
+  file: ./schema.graphql # link to the schema file
+dataSources:
+  - kind: cosmos
+    network: cosmoshub-4
+    source:
+      startBlock: 0 # Required for Cosmos, set this to 0 to start indexing from chain genesis
+    mapping:
+      apiVersion: 0.0.5
+      language: wasm/assemblyscript
+      blockHandlers:
+        - handler: handleNewBlock # the function name in the mapping file
+      eventHandlers:
+        - event: rewards # the type of the event that will be handled
```

Comment:
how are events identified on Cosmos, is it just a string (i.e. no signature with parameters, as on Ethereum?)